aspnetcore/fundamentals/minimal-apis/samples/routeGroups/RouteHandlers.cs

```diff
@@ -0,0 +1,66 @@
+using Data;
+using Microsoft.EntityFrameworkCore;
+
+public class RouteHandlers
+{
+    // get all todos
+    public static async Task<IResult> GetAllTodos(TodoGroupDbContext database)
+    {
+        var todos = await database.Todos.ToListAsync();
+        return TypedResults.Ok(todos);
+    }
+
+    // get todo by id
+    public static async Task<IResult> GetTodo(int id, TodoGroupDbContext database)
+    {
+        var todo = await database.Todos.FindAsync(id);
+        if (todo != null)
+        {
+            return TypedResults.Ok(todo);
+        }
+        return TypedResults.NotFound();
+    }
+
+    // create todo
+    public static async Task<IResult> CreateTodo(TodoDto todo, TodoGroupDbContext database)
+    {
+        var newTodo = new Todo
+        {
+            Title = todo.Title,
+            Description = todo.Description,
+            IsDone = todo.IsDone
+        };
+        await database.Todos.AddAsync(newTodo);
+        await database.SaveChangesAsync();
+        return TypedResults.Created($"/public/todos/{newTodo.Id}", newTodo);
```

Comment:
I think there is no `/public/todos/{newTodo.Id}` resource location. Should it be just `/todos/{newTodo.Id}`

Reply:
made changes to it 😅. thanks